src/package/packageVersionList.ts

```diff
@@ -35,6 +35,7 @@ const logger = Logger.childFromRoot('packageVersionList');
 export async function listPackageVersions(
   options: ListPackageVersionOptions
 ): Promise<QueryResult<PackageVersionListResult>> {
+  // TODO: replaces with connection.autoFetchQuery w/ tooling enabled
```

Comment:
still need to do this?

Reply:
Still waiting on review https://github.com/forcedotcom/sfdx-core/pull/665

Reply:
Replaced